core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -665,6 +673,15 @@ public FileInfo setXAttr(Map<String, byte[]> xAttr) {
     return this;
   }
 
+  /**
+   * @param cacheQuota the cache quota
+   * @return the updated {@link FileInfo}
+   */
+  public FileInfo setCacheQuota(CacheQuota cacheQuota) {
```

Comment:
Does a cache quota belong to a file? What happens if files have conflicting cache quotas?

Reply:
this is a workaround to cope with URIStatus